packages/auth/src/auth-service.ts

```diff
@@ -52,6 +53,10 @@ export class AuthService {
     return requests.createUser(user)
   }
 
+  async patchUserGroup(groupId: string, groupUpdateData: GroupUpdateData) {
```

Comment:
This seems too low level / implementation specific for the auth-service.
We shouldn't need to know about patch requests at this level and it shouldn't really be passing through to the underlying request with the URL path.

Is this being used anywhere?

Reply:
I moved this out mostly for testability, test coverage was failing here and this allows me to test to ensure this is not called, when the user already has the roile